packages/frint/src/App.ts

```diff
@@ -0,0 +1,410 @@
+import { Container, createContainer, Provider, resolveContainer } from 'frint-di';
+import { find, findIndex, get, omit } from 'lodash';
+import { BehaviorSubject, Observable } from 'rxjs';
```

Comment:
similar to the `lodash` comment above, please import them individually:

```js
import { BehaviorSubject } from 'rxjs/BehaviorSubject';
import { Observable } from 'rxjs/Observable';
```

Sidenote: the import paths for these basic types will change to `rxjs/create` in RxJS v6 (which is in alpha now)